### DISCUSSION TITLE
Is there a screen calibration function?

### DISCUSSION THREAD

**Discussion by MadyHeim** [eduardofilo/RG350_adam_image#229] [Ideas & Requests]
Hi, I am considering purchasing the rg300x console.
However, the console's screen is so blue.
I heard Adam Firmware is the best.
Does this firmware have a screen calibration function?
If not, do you have any plans to add the function?

**Comment by eduardofilo** [comment]
Hi. Actually Adam is not a firmware development but just a gathering of firmware, frontend and emulators. The question of how that device's display works and calibration possibilities should be directed to the firmware developers or user forums for that console. Although I think that thanks to ODbeta the display of the RG300X does indeed improve a lot.

**Reply by MadyHeim** [reply]
Thank you for your reply.
